merlin/schema/schema.py

```diff
@@ -286,7 +286,32 @@ def apply_inverse(self, selector) -> "Schema":
         return self.excluding(selector)
 
     def select_by_tag(self, tags: Union[Union[str, Tags], List[Union[str, Tags]]]) -> "Schema":
-        """Select matching columns from this Schema object using a list of tags
+        """Select all matching columns from this Schema object using a list of tags
```

Comment:
These docstrings might need an update to capture the difference between matching any tag and matching all tags. Maybe something like "Select columns from this Schema object that match all supplied tags"?